homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -36,7 +36,18 @@ async def async_setup_entry(
         timedelta(seconds=10),
         plenticore,
     )
-    for module_id, data_id, name, sensor_data, fmt in SENSOR_PROCESS_DATA:
+    module_id: str
+    data_id: str
+    name: str
+    sensor_data: dict[str, Any]
+    fmt: str
+    for (  # type: ignore[assignment]
```

Comment:
This component would benefit from refactoring to EntityDescription, but that's a bigger job and I think it's still useful to first take it out of `mypy` ignore list.